DVR/Tests/SessionUploadTests.swift

```diff
@@ -12,27 +12,35 @@ class SessionUploadTests: XCTestCase {
         return request
     }()
     let multipartBoundary = "---------------------------3klfenalksjflkjoi9auf89eshajsnl3kjnwal".UTF8Data()
-    lazy var testFile: NSURL = {
-        return NSBundle(forClass: self.dynamicType).URLForResource("testfile", withExtension: "txt")!
+    lazy var testFile: NSURL? = {
+        return NSBundle(forClass: self.dynamicType).URLForResource("testfile", withExtension: "txt")
```

Comment:
:-1: In tests, it's better to ! things so you can see the error right away durning the test.